oop/tests/unit/helpers/address.spec.js

```diff
@@ -0,0 +1,89 @@
+import { truncateAddressLines as Function } from "@/helpers/address.js";
```

Comment:
Question: Why are you aliasing the `truncateAddressLines` as `Function` here? I think it would be more clear to write out the actual function being called, no? Plus, if we ever need to add another named export function to the "address.js" file, then there wouldn't be any naming discrepancy between the two functions.

Reply:
That was because I found myself doing some copy pasting between files, and if I named it "Function," I only had to change one reference to the updated function instead of repeatedly throughout the document. It's the equivalent of "Component" in the Vue tests, since there's only one component being tested in each-- additional components required to test that test component are aliased appropriately, eg. "pageStateService".

If you want me to change that here, it would make sense for me to change it in all of the other files as well, which would take a little time but wouldn't be impossible.

Reply:
I'm ok with the name `Component` since it's technically a class that gets instantiated. I'd prefer it if you removed the alias of `Function` though since it's not a class or prototype, and it's not really a necessity to run the tests.

Sorry for creating more work for you.

Reply:
Oh, no worries. If it's just the Function ones, that's not too bad at all.

Fixed in f48b9723524b58aec1f88906cd7e255b0d30326b and bbf0aa29c542ada70680950d239b4b3b9c1c7c0a